rpa-manager/modules/ROOT/pages/processautomation-prepare-deployment-credential.adoc

```diff
@@ -0,0 +1,179 @@
+|===
+|*Action* |*Required Privileges*
+
+|Open the *Process Automation* module
+|Process Automation Open
+
+|Create and change global variables and credentials that can be used in the test phase
+|Global Variables Create for Test phase, Global Variables Edit for Test phase
+
+|Create and change global variables and credentials that can be used in the production phase
+|Global Variables Create for Production phase, Global Variables Edit for Production phase
+
+|Delete global variables and credentials
+|Global Variables Delete
+
+|===
+* If you have access to an Amazon Web Services (AWS) account and want to create AWS credentials for xref:rpa-builder::toolbox-aws-document-processing.adoc[RPA Builder - Document Processing with AWS], follow the instructions in https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html[AWS configuration basics^].
+* If you have access to a Microsoft Azure account and want to create OAuth credentials for xref:rpa-builder::toolbox-mail-operations-mail-session-outlook-with-oauth.adoc[RPA Builder - Mail Session (Outlook with OAuth)], create an application for Outlook, https://learn.microsoft.com/en-us/exchange/client-developer/exchange-web-services/how-to-authenticate-an-ews-application-by-using-oauth[authenticate an EWS application by using OAuth^], and https://learn.microsoft.com/en-us/exchange/client-developer/legacy-protocols/how-to-authenticate-an-imap-pop-smtp-application-by-using-oauth[authenticate an IMAP, POP, or SMTP connection using OAuth^].
+* If you want to use RPA Builder to xref:rpa-builder::toolbox-variable-handling-credentials-for-oauth.adoc#generating-an-oauth-refresh-token[generate a refresh token for your OAuth credentials], ask an administrator to xref:usermanagement-manage.adoc#assign-users-rpa-builder-access[assign you RPA Builder access].
+
+== Create a Credential
+
+Create credentials with which a bot can log in to an external application during process runs.
+
+To create a credential:
+
+. Open the *Credential Pool* view of the *Process Automation* module.
+. Click *Create*.
+. [[form-create-credential]] Complete the *Create a New Credential* form:
+* [[form-create-credential-useraffiliation]] *User affiliation*
++
+Click to choose users that may view, edit, and use the credential. Add yourself so that you can also view, edit, and use the credential.
+* *User group affiliation*
++
+Click to choose user groups whose users may view, edit, and use the credential.
+* *Phase affiliation*
++
+Phases in which the credential can be deployed. When you create or change the credential, your privileges determine which phases the user can select:
++
+** *Test*
++
+The credential can be deployed in the test phase.
++
+** *Production*
++
+The credential can be deployed in the production phase.
+
+* *Type*
++
+Type of the global credential. The type of credential determines where you can use it. The original value defined in RPA Builder is typed. Select one of the following types:
```

Comment:
What does "typed" mean in this context? 

Does it mean that the default value appears?

Reply:
Typed is an expression from computer science. 
See e.g. paragraph Typing in https://en.wikipedia.org/wiki/Variable_(computer_science)

Typed in this context means that the RPA developer assigned a type to an activity parameter. This type decides what values the variable can assume. For example, if the variable is of type integer, the user cannot give it a value of type float or type string. He can assign '42', but not '42.0' or 'forty-two'. This is important because not all operations you can think of are possible with all types of variables. Mathematical operations like addition and substraction can be performed on ints and floats but not on strings. (Some programming languages allow this. For example in python, you can compute 'bla'*3 which results in 'blablabla'). But RPA Builder can only use typed variables.

If 'typed' is a term used only in computer science and is not commonly understood, I could replace it with 'has a type'. 

Would that help?

Reply:
Or maybe use the term 'data type' like in https://docs.mulesoft.com/rpa-builder/6.1/toolbox-variable-handling-activity-parameters?

Reply:
I think, if your audience will understand it, then use "typed". It just wasn't clear to me what this meant. My first thought is, what is the type of the original value?

Reply:
I created a story for that: [Explain variable types for non-coders [RPA Manager - Process Automation - Prepare Deployment]](https://gus.lightning.force.com/a07EE00001OQ596YAD)